CMakeLists.txt

```
@@ -91,6 +103,7 @@ set(_linker_options ${MKL_UMATH_LINKER_OPTIONS})
 set(_trgt mkl_umath_loops)
 add_library(${_trgt} SHARED "mkl_umath/src/mkl_umath_loops.c")
 set_target_properties(${_trgt} PROPERTIES CMAKE_POSITION_INDEPENDENT_CODE ON)
+set_target_properties(${_trgt} PROPERTIES  C_STANDARD 99)
```

I'd propose to combine, like:
```suggestion
set_target_properties(${_trgt}
  PROPERTIES
    CMAKE_POSITION_INDEPENDENT_CODE ON
    C_STANDARD 99)
```